tools/dexnode/src/index.ts

```diff
@@ -6,264 +6,414 @@ import semver from "semver";
 import path from "path";
 import fs from "fs";
 import os from "os";
+import Registry from "winreg";
 
 declare global {
     var Deno: { version?: { v8?: string; }; } | undefined;
 }
 
-const NULL_DEVICE =
-    process.platform === "win32" ? "\\\\.\\NUL" :
-    canWriteToDevNull() ? "/dev/null" :
-    null;
+const chromeDefaultPaths: Partial<Record<NodeJS.Platform, string[]>> = {
+    win32: ["C:/Program Files/Google/Chrome/Application/chrome.exe", "C:/Program Files (x86)/Google/Chrome/Application/chrome.exe"],
+    darwin: ["/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"],
+    linux: ["/usr/bin/google-chrome"],

```

Comment:
It's possible that this is `/usr/bin/google-chrome-stable` too, but I know that pprof just has a list of things to check on PATH (which I also work around by aliasing something with one of these names on PATH).

https://github.com/google/pprof/blob/eeec1cb781c311df467fabdc8c384fa52e91bc65/internal/driver/commands.go#L354

Reply:
I'll add both forms. I'm not adding dev/canary channels at this time since you can override the executable path using `--exec-path`.